### PR TITLE
Fix/improve startup-shutdown.md

### PR DESCRIPTION
**Pull request by mglae**
- kodi-autostart.service is depending on network-online.target, network is up.
- Document introduction of autostop.sh

### CODE REVIEW

File: configuration/startup-shutdown.md [hunk 1]
```diff
@@ -27,6 +29,8 @@ nano /storage/.config/autostop.sh
 
 Like the `autostart.sh` script you can place any commands in the script, and they will block the shutdown process for up to 5 minutes and then will be terminated automatically.
 
+`autostop.sh` is avalable as of LibreELEC 10.0.2 or LibreELEC 11 nightlys 2022-02-04.
```
Comment (chewitt) [inline configuration/startup-shutdown.md:32]: s/nightlys/nightlies

File: configuration/startup-shutdown.md [hunk 2]
```diff
@@ -35,7 +39,7 @@ The `shutdown.sh` script run during the shutdown process. It does not exist by d
 nano /storage/.config/shutdown.sh
 ```
 
-Unlike the `autostart.sh` script, the `shutdown.sh` script should follow the following template to ensure commands are executed during the correct event, i.e. you can run different commands for a `reboot` event to `halt` event, or use the `*` as a catch-all.
+Unlike the `autostart.sh` script the network stack will not be up when the script runs and the `shutdown.sh` script should follow the following template to ensure commands are executed during the correct event, i.e. you can run different commands for a `reboot` event to `halt` event, or use the `*` as a catch-all.
```
Comment (chewitt) [inline configuration/startup-shutdown.md:42]: remove 'not' ? .. surely the network _is_ up when shutdown.sh runes?

File: configuration/startup-shutdown.md (hunk 1 above)
Comment (mglae) [reply configuration/startup-shutdown.md]: Done

**Comment by mglae** [comment]
Added fix for `case .. esac` sample code and documentation, see https://forum.libreelec.tv/thread/25509-le-10-2-rpi4-shutdown-sh-script-not-working/

**Review by chewitt — APPROVED** [review]

**Pull request merged**